packages/components/src/components/ViewYAML/ViewYAML.js

```diff
@@ -15,20 +15,23 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import jsYaml from 'js-yaml';
 import classNames from 'classnames';
+import { Prism as SyntaxHighlight } from 'react-syntax-highlighter';
 
 const ViewYAML = props => {
-  const { className, dark, resource } = props;
+  const { className, dark, resource, enableSyntaxHighlighting = false } = props;
 
   return (
-    <div
+    <SyntaxHighlight
       className={classNames('bx--snippet--multi', className, {
-        'tkn--view-yaml--dark': dark
+        'tkn--view-yaml--dark': dark,
+        'tkn--view-yaml--enable-syntax-highlighting': enableSyntaxHighlighting
       })}
+      codeTagProps={{}}
```

Comment:
from the docs:

> `codeTagProps` - props that will be spread into the `<code>` tag that is the direct parent of the highlighted code elements. Useful for styling/assigning classNames.

Seems like it should be safe to omit this. Did you run into problems if you left it out?

Reply:
@AlanGreene I found that if we omit this. SyntaxHighlight will add some inline style to the element. It can cause we cannot be styling syntax highlight in sass world.

Reply:
This is the one issue if I doesn't add `codeTagProps: {{}}`:

<img width="1299" alt="Screen Shot 2564-06-09 at 22 47 37" src="https://user-images.githubusercontent.com/484530/121387208-b67d8200-c974-11eb-8954-c4b9360cc240.png">

The syntax highlighter add some inline styling even set `useInlineStyles` to `false`.